package.json

```diff
@@ -60,16 +60,15 @@
       }
     ],
     "directories": {
-      "output": "release",
-      "app": "source"
+      "output": "release"
     },
     "afterPack": "./scripts/afterPack.js",
     "afterSign": "./scripts/afterSign.js",
     "mac": {
       "category": "public.app-category.productivity",
       "target": "dmg",
       "artifactName": "Zettlr-${version}-${arch}.${ext}",
-      "icon": "resources/icons/icns/icon.icns",
+      "icon": "resources/icons/icon.icns",
```

Comment:
One super important question I just realized: Electron forge also requires the icons; so does this mean that the prepackaged app produced by electron-forge contains ALL of Windows 64bit, Windows 32 bit, Windows 64bit ARM? This might actually be a problem, and we need to check that!

Reply:
Tobi: Drop 32, electron-forge-maker-nsis for future